src/Cms/PageActions.php

```diff
@@ -768,11 +777,15 @@ public function resortSiblingsAfterUnlisting(): bool
     }
 
     /**
+     * @deprecated 3.5.0 Use `Page::changeSort()` instead
```

Comment:
We already have `$page->changeStatusToListed($position)`, which does exactly what the `changeSort()` method would do. So I'd point directly to this method here.

Reply:
I would keep the `changeStatusTo...` methods protected to have consistency. Those are not meant to be called directly, instead use `$this->changeStatus('listed', $position)` then.